src/jit/compiler.h

```diff
@@ -8588,8 +8588,9 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
         unsigned  compArgsCount;     // Number of arguments (incl. implicit and     hidden)
 
 #if FEATURE_FASTTAILCALL
-        size_t compArgStackSize; // Incoming argument stack size in bytes
-#endif                           // FEATURE_FASTTAILCALL
+        size_t compArgStackSize;   // Incoming argument stack size in bytes
+        bool compHasMultiSlotArgs; // Caller has >8 byte sized struct parameter
```

Comment:
I much prefer this "MultiSlot" nomenclature to "Multireg" as in `hasMultiregStructArgs` in `fgMorphArgs`. I think we should change that one to `hasMultiSlotStructArgs` (since it must be set for multi-slot structs even if they're not enregistered), but as I recall there were objections to changing that in one of my earlier refactors, probably because there were already so many changes.